Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/ObjCStubs.kt

```diff
@@ -585,3 +606,22 @@ private fun NativeCodeBuilder.genMethodImp(
     return functionName
 }
 
+private fun genProtocolGetter(
+        stubGenerator: StubGenerator,
+        nativeBacked: NativeBacked,
+        protocol: ObjCProtocol
+): String {
+    val functionName = "kniprot_" + stubGenerator.pkgName.replace('.', '_') + stubGenerator.nextUniqueId()
+
+    val builder = NativeCodeBuilder(stubGenerator.simpleBridgeGenerator.topLevelNativeScope)
+
+    with(builder) {
+        out("Protocol* $functionName() {")
```

Comment:
Multiline string?

Reply:
`out` isn't supposed to take multiline strings.